sources/org.osbuild.curl

```diff
@@ -123,7 +129,15 @@ def fetch(url, checksum, directory):
             # url must follow options
             curl_command.append(url_path)
 
-            curl = subprocess.run(curl_command, encoding="utf-8", cwd=tmpdir, check=False)
+            if proxy:
+                # Continue to inherit environment
+                env = os.environ.copy()
```

Comment:
Extremely minor nitpick: maybe do the `env = os.environ.copy()` at the top of the scope? That way, in case we add other things in addition to `proxy` we don't accidentally end up overwriting the `env` var rather than appending to it. That's a pretty theoretical concern, so also fine as is.

Reply:
Sure, doesn't hurt to prevent future breakage ;)